index.html

```diff
@@ -0,0 +1,232 @@
+                    </ul>
+                </div>
+                <div class="button-m">
+                    <button type="button">See Project</button>
+                </div>
+            </div>
+            <div class="back-image-file-3 all-image-pro mid-end"></div>
+        </article>
+    </section>
+    <section class="about-me" id="about-1">
+        <article>
+            <div class="about-me-page">
+                <h1 class="header-1">About Myself</h1>
+                <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <div class="let-s">
+                    <h1>LET'S CONNECT</h1>
+                    <div class="vector">
+                        <ul>
+                            <li><a href="https://github.com/girma336" class="twitter"></a></li>
+                            <li><a href="#" class="linkedin"></a></li>
+                            <li><a href="#" class="medium"></a></li>
+                            <li><a href="#" class="github"></a></li>
+                            <li><a href="#" class="hand"></a></li>
+                        </ul>
+                    </div>
+                </div>
+                <div class="button-m">
+                    <button type="button">See Project</button>
+                </div>
+            </div>
+            <div class="lists-of-frame">
+                <ul>
+                    <li class="language"><h1>Language</h1>
+                        <div class="up-down">
+                        <span  id="odd-span"  class="glyphicon glyphicon-chevron-right"></span></p>
+                        <span  id="new-span"  class="glyphicon glyphicon-chevron-down"></span></p>
+                        </div>
+                        
+                    
+                    <ul class="hovers">
+                        <li>
+                            <span class="list-1"></span>
+                             <p>JavaSript</p>
+                        </li>
+                        <li>  
+                            <span class="list-2"></span>                                
+                            <p>Htlm</p>
+                        </li>
+                        <li>
+                            <span class="list-3"></span>                             
+                            <p>Css</p>
+                        </li>
+                    
+                    </ul></li>
+                    <li class="lang"><h1>Frameworks</h1>
+                    <span id="span" class="glyphicon glyphicon-chevron-right"></span></p>    
+                    </li>
+                    <li class="lag">
+                    <h1>Skills</h1>
+                    <span id="span"  class="glyphicon glyphicon-chevron-right"></span></p>    
+                    </li>
+                </ul>
+            </div>  
```

Comment:
- [ Optional ] Please make sure that mobile design still looks like expected Just Like The Figma design, I recommend you to adjust alignment for the arrows just like the figma design. 

Your List of frames design Look likes this : 
 
![Capture](https://user-images.githubusercontent.com/93598189/194413015-55f0c3a1-c1a8-4749-b85a-dee8b58318ab.PNG)


The Figma design Look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/194410897-8a98f643-e0af-46ee-836c-8f975ac87232.PNG)